server/src/main/java/org/apache/cassandra/sidecar/acl/authorization/AuthorizationWithAdminBypassHandler.java

```diff
@@ -55,13 +53,9 @@ public void handle(RoutingContext ctx)
         }
 
         List<String> identities = extractIdentities(ctx.user());
-        if (identities.isEmpty())
```

Comment:
With sidecar supporting multiple authenticators, we shouldn't mandate identities for authorization